release-notes/10.0/preview/preview5/aspnetcore.md

```diff
@@ -0,0 +1,165 @@
+# ASP.NET Core in .NET 10 Preview 5 - Release Notes
+
+Here's a summary of what's new in ASP.NET Core in this preview release:
+
+- [Configure custom security descriptors for HTTP.sys request queues](#configure-custom-security-descriptors-for-httpsys-request-queues)
+- [Validation resolver APIs marked as experimental](#validation-resolver-apis-marked-as-experimental)
+- [Support for generating OpenAPI 3.1](#support-for-generating-openapi-31)
+- [OpenAPI metadata from XML doc comments](#openapi-metadata-from-xml-doc-comments)
+- [Add a Not Found page using the Blazor Router](#add-a-not-found-page-using-the-blazor-router)
+- [Blazor metrics and tracing](#blazor-metrics-and-tracing)
+
+ASP.NET Core updates in .NET 10:
+
+- [What's new in ASP.NET Core in .NET 10](https://learn.microsoft.com/aspnet/core/release-notes/aspnetcore-10.0) documentation.
+- [Breaking changes](https://docs.microsoft.com/dotnet/core/compatibility/10.0#aspnet-core)
+- [Roadmap](https://github.com/dotnet/aspnetcore/issues/59443)
+
+## Configure custom security descriptors for HTTP.sys request queues
```

Comment:
Get rid of the "Why this matters" and "Hot to use" headings and connect with prose instead.